src/themes/icdcLight.js

```diff
@@ -1,394 +0,0 @@
-import tinycolor from 'tinycolor2';
```

Comment:
Are we removing this file?

Reply:
No we are not. Its is missing in this PR. Will add and update the pr.

Reply:
icdcLight was initially part of old PR. Which was separated to different PR and merged to "bento". Will update this PR accordingly.